src/components/WebexInMeeting/WebexInMeeting.scss

```diff
@@ -4,7 +4,6 @@ $shadow-spread: -0.125rem;
 $shadow-color: rgba(0, 0, 0, 0.2);
 
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-in-meeting {
-  background: black;
```

Comment:
remove the blank line remaining and please add this in legacy.scss, but cancel it in theme-dark.scss and theme-light.scss (background: transparent).